plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/BaseClickHouseTypeMapping.java

```diff
@@ -455,6 +488,18 @@ public void testClickHouseChar()
                 .addRoundTrip("Nullable(char(10))", "NULL", VARBINARY, "CAST(NULL AS varbinary)")
                 .addRoundTrip("Nullable(char(10))", "'text_a'", VARBINARY, "to_utf8('text_a')")
                 .addRoundTrip("Nullable(char(1))", "'😂'", VARBINARY, "to_utf8('😂')")
+                .addRoundTrip("Nullable(char(255))", "''", VARBINARY, "X''")
+                // low-cardinality
+                .addRoundTrip("LowCardinality(char(10))", "'text_a'", VARBINARY, "to_utf8('text_a')")
+                .addRoundTrip("LowCardinality(char(255))", "'text_b'", VARBINARY, "to_utf8('text_b')")
+                .addRoundTrip("LowCardinality(char(5))", "'攻殻機動隊'", VARBINARY, "to_utf8('攻殻機動隊')")
+                .addRoundTrip("LowCardinality(char(32))", "'攻殻機動隊'", VARBINARY, "to_utf8('攻殻機動隊')")
+                .addRoundTrip("LowCardinality(char(1))", "'😂'", VARBINARY, "to_utf8('😂')")
+                .addRoundTrip("LowCardinality(char(77))", "'Ну, погоди!'", VARBINARY, "to_utf8('Ну, погоди!')")
+                // low-cardinality nullable
+                .addRoundTrip("LowCardinality(Nullable(char(10)))", "NULL", VARBINARY, "CAST(NULL AS varbinary)")
+                .addRoundTrip("LowCardinality(Nullable(char(10)))", "'text_a'", VARBINARY, "to_utf8('text_a')")
+                .addRoundTrip("LowCardinality(Nullable(char(1)))", "'😂'", VARBINARY, "to_utf8('😂')")
```

Comment:
These new tests are unrelated to "Enable varchar equality predicate pushdown for ClickHouse connector", right? Please extract the changes as a preparatory commit.

Reply:
I copied these checks from respective postgres unit tests, to bring them to some kind of sync with regard to varchar/string support. So they're related to the PR. 

I can drop them if you think they're excessive.

Reply:
Please separate a commit.

Reply:
Sorry for stupid question, what do you mean by "preparatory commit"? A separate PR with new unit tests only? Or a separate commit within this PR?

Reply:
I mean the latter, "a separate commit within this PR".

Reply:
Done, please see https://github.com/trinodb/trino/pull/21557/commits/c15a930fa638604e4bf0bfe55ba8435ef2ced9d2